[docs] Update API signatures (#5677)
DevExpress/testcafe: docs/articles/documentation/reference/test-api/testcontroller/switchtoparentwindow.md

# t.switchToParentWindow method

> Activates the window that launched or was active during the launch of the active window. Can be chained with other `TestController` methods.

Reviewer: ```suggestion
Activates the window that launched or was active when the active window was launched. Can be chained with other `TestController` methods.
```

Reviewer: "Activates" "active"
